### ISSUE TITLE
Installer getting flagged as Malicious on VirusTotal and Hybrid-Analysis

### ISSUE THREAD

**Issue by Silver-Streak** [LibreSprite/LibreSprite#80]
I shared this with a community I'm a member of, and they brought up concerns that it was showing as a virus by some online resources.

I did some checking with the windows installer and am showing similar hits, although low on the total:

Virus Total: https://www.virustotal.com/gui/file/0b97559ddb9979f67836941b52caadafba37c22b639aa8bc0643cd17916a5324/detection

Hybrid Analysis Results: https://www.hybrid-analysis.com/sample/0b97559ddb9979f67836941b52caadafba37c22b639aa8bc0643cd17916a5324

and from their VM test:
https://www.hybrid-analysis.com/sample/0b97559ddb9979f67836941b52caadafba37c22b639aa8bc0643cd17916a5324/5f57857e07ff291774537e6f

Looks like it could be related to Inno Setup, or something else.

**Comment by MintStudios** [comment]
Maybe.. I'm gonna check the actual .exe file (not the installer)

**Comment by MintStudios** [comment]
libresprite.exe is clean for virustotal and hybrid analysis. 

https://www.virustotal.com/gui/file/a995e7b09799d8390b2484cc6e29e34be8d15a6a7d59c86ec03c7c93b16a00a4/detection

https://www.hybrid-analysis.com/sample/a995e7b09799d8390b2484cc6e29e34be8d15a6a7d59c86ec03c7c93b16a00a4

It might be the other .exe files, in the program, such as gen.exe or unins000.exe. I'll look into those.

**Comment by MintStudios** [comment]
Alright, I have scanned gen.exe and unins000.exe. Both have some red flags. Gen.exe is safe on virustotal, but has is detected by metadefender. unins000.exe is detected on virustotal.

**Comment by Silver-Streak** [comment]
@MintStudios thanks for digging into it. Amy ideas what's causing the flags, and/or is it addressable?

**Comment by ssokolow** [comment]
> I think maybe some programs that contain malware use innosetup, and that might have made the scanners think that LibreSprite is also malware.

Yeah. Last I checked, some of the scanners on VirusTotal were calling the official installer for the [NSIS](https://en.wikipedia.org/wiki/Nullsoft_Scriptable_Install_System) compiler malware, and, as Wikipedia points out, NSIS has big-name users like Amazon.com, Dropbox, Google, Ubisoft, FL Studio, BitTorrent, and McAfee.

Likewise, when I built some [legal RAR test files](https://github.com/ssokolow/rar-test-files), I found that some scanners just [like to assume](https://github.com/ssokolow/rar-test-files#my-virus-scanner-reports-malware-in-these-files) that the RAR self-extractor stub is malware.

**Comment by jjhaggar** [comment]
ClamWin throws a warning for the installer too... 
![image](https://user-images.githubusercontent.com/4078392/133530491-8f005d51-67fd-442f-939c-7b9184e44ee4.png)

...But not for the installed files
![image](https://user-images.githubusercontent.com/4078392/133530527-caf92f50-9268-45b5-b491-c820dea2688c.png)


Also, is this issue related to (or the same issue as) #64 ?

**Comment by Zughy** [comment]
Adding relevant comment from @maujin111 in #64 (which was a duplicate, thanks @jjhaggar )

"I did a little research and found [these](https://stackoverflow.com/questions/252226/signing-a-windows-exe-file) options, some seem viable and interesting."

**Comment by godofgrunts** [comment]
You could also look into [sigstore](https://www.sigstore.dev/) for signing the binaries.

**Comment by Batatexy** [comment]
paia isso ai em
